tests/views/resource_tests.py

```diff
@@ -354,3 +354,21 @@ def test_get_related_resource(self):
         ret = reference_resource.get_related_resources(user=user)
         relationship = ret["resource_relationships"][0]["relationshiptype_label"]
         self.assertEqual(relationship, en_preflabel)
+
+    def test_resource_report_good(self):
+        self.client.login(username="admin", password="admin")
+        url = reverse(
+            "resource_report", kwargs={"resourceid": self.resource_instance_id}
+        )
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_resource_report_bad(self):
```

Comment:
Because `test_resource_report_good` tests if the status is 200, that name makes sense. In this case though, I think what's being tested isn't whether the response is bad (which I would take to mean a 500) but rather whether it's missing. So I think more intuitive name would be `test_resource_report_for_missing_resource` or something like that.